storybookjs/storybook
# Core: Add option to use webpack filesystem cache

Issue: #16072 

## What I did
I have this feature working, but I am not confident that I have done it in the best way. I am open to feedback, or allowing someone else to take over this feature.

## How to test
* I have not included tests because the webpack builders do not have any existing tests.
* There are no examples in the repo that use webpack5, so I cannot show this in an example. I tested this change manually by updating web-components-kitchen-sink to use webpack5 and adding this config option. I confirmed that a folder `node_modules/.cache/webpack` is created and includes cache files.
* I'm not sure where documentation for this would go. I only see mention of 'webpack5' in the migration documentation, but this is not a breaking change, so I don't think this belongs there
